pkg/flowaggregator/exporter/ipfix_test.go

```diff
@@ -288,3 +291,97 @@ func TestIPFIXExporter_sendRecord_Error(t *testing.T) {
 
 	assert.Error(t, ipfixExporter.AddRecord(mockRecord, false))
```

Comment:
The TestIPFIXExporter_sendRecord_Error test function is testing the AddRecord function with a mock record and asserting that it returns an error. However, it is not clear what error is being expected. It would be better to add a specific error message to the assertion in the 3rd argument